tests/posix_time/main.c

```diff
@@ -23,22 +24,22 @@
 
 int main(void)
 {
-    puts("usleep 1 x 1000*1000");
-    for (int i = 0; i < 10; i++) {
-        useconds_t us = i*1000u*1000u;
-        printf("calling usleep(%u)\n", (unsigned int) us);
+    puts("Please hit any key and then ENTER to continue");
+    getchar();
```

Comment:
why? This looks somewhat ugly (and complicated), IIRC testrunner resets the device after successfully establishing a terminal/serial connection.

Reply:
To sync the `testrunner`. 

> IIRC testrunner resets the device after successfully establishing a terminal/serial connection.

yes, but it isn't synchronous i.e. if `pexpect` picks up an output it might already there for quite a while, so output enough isn't enough to wait. We need to wait for some signal from the expect script. stdin is the easiest way to do that. This method was already used in `tests/xtimer_usleep`

Reply:
I see - leave as is for now ...